scripts/west_commands/runners/openocd.py

```diff
@@ -47,8 +47,9 @@ def __init__(self, cfg, pre_init=None, reset_halt_cmd=DEFAULT_OPENOCD_RESET_HALT
                     search_args.append('-s')
                     search_args.append(path.dirname(i))
 
-        for p in cfg.openocd_search:
-            search_args.extend(['-s', p])
+        if cfg.openocd_search is not None:
```

Comment:
could probably be simplified to
```suggestion
        if cfg.openocd_search:
```

Reply:
I'm trying follow convention on the file.